release_notes/v0.9.md

```diff
@@ -0,0 +1,302 @@
+# OPEA Release Notes v0.9
+## What’s New in OPEA v0.9
+
+- Broaden functionality
+    - Initialize two Agent examples
+    - Support the experimental Fine-tuning microservice
+    - Use Llama-Guard-2-8B as default Guardrails model and bge-large-zh-v1.5 as default mosec embedding & reranking model
+    - Provides telemetry functionalities for metrics and tracing using Prometheus, Grafana, and Jaeger
+    - Add a benchmark script to assess the performance of the examples
+
```

Comment:
- Re-organized documentation towards making developer and user experience smoother. Introduced the use of Sphinx to build and publish docs.

Reply:
@dbkinder and @tomlenth please see if you would like to add more above. We keep highlights pretty brief.

@kevinintel would you kindly pull into the details section the activity in the doc repository.

Reply:
add githubio url, but the title of the repo looks not formal. David can think how to add it.